CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ This is an example of a brief overview of the _Major_ or _Minor_ version changes
 
 ### **BREAKING CHANGES**
 
+#### Tooltip component
```

Comment:
We should also add a note for the non-breaking change of extra positioning options for `Tooltip`